presenters/trial.js

```diff
@@ -58,6 +58,17 @@ function decorateDocuments(documents) {
   return documentsByCategory;
 }
 
+function decorateFDADocuments(documents) {
+  const docs = _.filter(documents, (doc) => doc.source_id === 'fda');
+  _.map(docs, (doc) => {
+    if (doc.fda_approval) {
+      // eslint-disable-next-line no-param-reassign
+      doc.application_id = doc.fda_approval.fda_application.id;
+    }
+  });
```

Comment:
Good suggestions, I somehow overlooked that part of the code while being focused on the "make it pretty" side of things. Thanks for the suggestion, I'll push an update in a few minutes.